$OpenBSD: patch-akode_plugins_ffmpeg_decoder_Makefile_am,v 1.1 2011/04/03 06:37:29 ajacoutot Exp $

C++ code needs to define __STDC_CONSTANT_MACROS to pull in parts of
stdint.h (needed by graphics/ffmpeg).

--- akode/plugins/ffmpeg_decoder/Makefile.am.orig	Thu Mar 31 23:32:40 2011
+++ akode/plugins/ffmpeg_decoder/Makefile.am	Thu Mar 31 23:34:47 2011
@@ -4,5 +4,7 @@ lib_LTLIBRARIES	= libakode_ffmpeg_decoder.la
 
 libakode_ffmpeg_decoder_la_SOURCES = ffmpeg_decoder.cpp
 
+AM_CPPFLAGS = -D__STDC_CONSTANT_MACROS
+
 libakode_ffmpeg_decoder_la_LDFLAGS = -avoid-version -no-undefined $(AVCODEC_LDFLAGS) $(AVFORMAT_LDFLAGS)
 libakode_ffmpeg_decoder_la_LIBADD = ../../lib/libakode.la $(AVCODEC_LIBADD) $(AVFORMAT_LIBADD)
